examples/slu/slurp/eval_utils/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .evaluation.metrics import ErrorMetric
```

Comment:
examples directory are not supposed to be python modules. Dont treat them as such with __init__.

Reply:
Basically refactor to not have to import from files, treat each script as a self contained script and don't treat subdirs in examples as python modules